sdk/internal/internal.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal // import "go.opentelemetry.io/sdk/internal"
```

Comment:
about `internal` package https://golang.org/doc/go1.4#internalpackages and should we use `internal` name? thanks!

Reply:
The exported apis in the this file is meant only for the sdk. So it is under internal directory. Are you suggesting that we move this file to sdk/trace and make the functions unexported?